api/types/usertoken.go

```diff
@@ -98,6 +103,16 @@ func (u *UserTokenV3) SetURL(url string) {
 	u.Spec.URL = url
 }
 
+// SetVerifiedMFADeviceID sets the field for the ID of a validated MFA device.
+func (u *UserTokenV3) SetVerifiedMFADeviceID(deviceId string) {
+	u.Spec.VerifiedMFADeviceID = deviceId
```

Comment:
```suggestion
func (u *UserTokenV3) SetVerifiedMFADeviceID(deviceID string) {
	u.Spec.VerifiedMFADeviceID = deviceID
```

https://github.com/golang/go/wiki/CodeReviewComments#initialisms